### ISSUE TITLE
Error on http://swordandsandals.top/swords-and-sandals-2-unblocked/

### ISSUE THREAD

**Issue by Fkdenzocker323** [ruffle-rs/ruffle#13620]
# Error Info
Error name: Error
Error message: panicked at /home/runner/.cargo/git/checkouts/wgpu-53e70f8674b08dd4/49d16f7/wgpu-hal/src/gles/device.rs:705:52:
called `Result::unwrap()` on an `Err` value: "Unable to create texture object"
Error stack:
```
Error: panicked at /home/runner/.cargo/git/checkouts/wgpu-53e70f8674b08dd4/49d16f7/wgpu-hal/src/gles/device.rs:705:52:
called `Result::unwrap()` on an `Err` value: "Unable to create texture object"
    at U.n.wbg.__wbg_new_d258248ed531ff54 (https://unpkg.com/@ruffle-rs/ruffle/core.ruffle.b3970bebcf9c117d3dbf.js:1:53228)
    at core::ops::function::FnOnce::call_once{{vtable.shim}}::h307f95dbfd4e7fe6 (wasm://wasm/032955f2:wasm-function[9716]:0x98c1a1)
    at std::panicking::rust_panic_with_hook::h3aa054d35a0817d7 (wasm://wasm/032955f2:wasm-function[6377]:0x890b79)
    at core::panicking::panic_fmt::h3e1dd3d08288569e (wasm://wasm/032955f2:wasm-function[9525]:0x98716a)
    at core::result::unwrap_failed::h8b3db0f11171b57b (wasm://wasm/032955f2:wasm-function[7993]:0x90d2a3)
    at <T as wgpu::context::DynContext>::device_create_texture::he876a6b2d583c4cf (wasm://wasm/032955f2:wasm-function[7103]:0x8c95d8)
    at wgpu::Device::create_texture::h1488ca4a9eae3889 (wasm://wasm/032955f2:wasm-function[5425]:0x84078d)
    at <ruffle_render_wgpu::backend::WgpuRenderBackend<T> as ruffle_render::backend::RenderBackend>::create_empty_texture::h6bc32d323fa51381 (wasm://wasm/032955f2:wasm-function[5242]:0x82f3e5)
    at ruffle_core::display_object::render_base::h3a668d845d026b36 (wasm://wasm/032955f2:wasm-function[598]:0x149105)
    at <ruffle_core::display_object::DisplayObject as ruffle_core::display_object::TDisplayObject>::render::haf72a4b5fbd8e829 (wasm://wasm/032955f2:wasm-function[6977]:0x8b7549)
```

# Player Info
Allows script access: true
Renderer: wgpu
Adapter Backend: Gl
Adapter Name: "ANGLE (NVIDIA, NVIDIA GeForce GTX 1650 SUPER Direct3D11 vs_5_0 ps_5_0, D3D11)"
Adapter Device Type: Other
Adapter Driver Name: ""
Adapter Driver Info: ""
Enabled features: Features(TEXTURE_ADAPTER_SPECIFIC_FORMAT_FEATURES | PUSH_CONSTANTS | SHADER_UNUSED_VERTEX_OUTPUT)
Available features: Features(TEXTURE_COMPRESSION_BC | CLEAR_TEXTURE | MULTIVIEW)
Current limits: Limits { max_texture_dimension_1d: 16384, max_texture_dimension_2d: 16384, max_texture_dimension_3d: 2048, max_texture_array_layers: 256, max_bind_groups: 4, max_bindings_per_bind_group: 1000, max_dynamic_uniform_buffers_per_pipeline_layout: 8, max_dynamic_storage_buffers_per_pipeline_layout: 0, max_sampled_textures_per_shader_stage: 16, max_samplers_per_shader_stage: 16, max_storage_buffers_per_shader_stage: 0, max_storage_textures_per_shader_stage: 0, max_uniform_buffers_per_shader_stage: 11, max_uniform_buffer_binding_size: 16384, max_storage_buffer_binding_size: 0, max_vertex_buffers: 8, max_buffer_size: 268435456, max_vertex_attributes: 16, max_vertex_buffer_array_stride: 255, min_uniform_buffer_offset_alignment: 256, min_storage_buffer_offset_alignment: 256, max_inter_stage_shader_components: 60, max_compute_workgroup_storage_size: 0, max_compute_invocations_per_workgroup: 0, max_compute_workgroup_size_x: 0, max_compute_workgroup_size_y: 0, max_compute_workgroup_size_z: 0, max_compute_workgroups_per_dimension: 0, max_push_constant_size: 96 }
Surface quality: high
Surface samples: 4
Surface size: Extent3d { width: 550, height: 400, depth_or_array_layers: 1 }

# Page Info
Page URL: http://swordandsandals.top/swords-and-sandals-2-unblocked/
SWF URL: http://swordandsandals.top/wp-content/uploads/swordsandsandals2-hacked.swf

# Browser Info
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36 OPR/102.0.0.0
Platform: Win32
Has touch support: false

# Ruffle Info
Version: 0.1.0
Name: nightly 2023-10-19
Channel: nightly
Built: 2023-10-19T00:23:16.593Z
Commit: fd4d6921d851e788b1386affae1498e46cda939c
Is extension: false

# Metadata
width: 700
height: 525
frameRate: 30
numFrames: 1
swfVersion: 8
backgroundColor: #FFFFFF
isActionScript3: false
uncompressedLength: 760

**Comment by evilpie** [comment]
Duplicate of https://github.com/ruffle-rs/ruffle/issues/11852
